lib/editor/util/map.js

```diff
@@ -44,11 +46,24 @@ type R5Response = {
   }>
 }
 
-export const stopIsOutOfBounds = (stop: GtfsStop, bounds: any) => {
-  return stop.stop_lat > bounds.getNorth() ||
-    stop.stop_lat < bounds.getSouth() ||
-    stop.stop_lon > bounds.getEast() ||
-    stop.stop_lon < bounds.getWest()
+export const stopIsOutOfBounds = (stop: GtfsStop | GtfsLocation, bounds: any) => {
+  if (stop.location_id) {
+    const {location_shapes: locationShapes} = stop
+
+    const PADDING = 0.15
+    return locationShapes.some(locationShapePt =>
+      locationShapePt.geometry_pt_lat > bounds.getNorth() + PADDING ||
+      locationShapePt.geometry_pt_lat < bounds.getSouth() - PADDING ||
+      locationShapePt.geometry_pt_lon > bounds.getEast() + PADDING ||
+      locationShapePt.geometry_pt_lon < bounds.getWest() - PADDING
+    )
+  // Check for stop_id to appease flow
+  } else if (stop.stop_id) {
```

Comment:
Hold on, how is this type check working here but not elsewhere?

Reply:
Yes. I have spent good time pondering this.

Reply:
I'm guessing datatools is so complicated it's causing flow bugs